stories/settings.stories.js

```diff
@@ -35,13 +35,13 @@ import { __ } from '@wordpress/i18n';
jHH7njg0f6q5z\\\/wA+P\\\/Jo\\\/wDyRL4myd71v\\\/JZ\\\/wCRJH+2f8FkiVE8YqIwMKBpd5gD0\\\/1NH+qmc\\\/8APj\\\/yaP8A8kP\\\/AFnyhae2\\\/wDJZ\\\/5Dv+G1PgyP+Zz\\\/APKZef8Axmj\\\/AFUzn\\\/nx\\\/wCTR\\\/8Akg\\\/1nyn\\\/AJ\\\/f+Sy\\\/yPnf\\\/goP+1D8M\\\/iF+yB8QfD\\\/AIf8Tf2hq94lkILf7Bcx79t9bu3zPGFGFVjye3rXNieHszwNGWIxFK0Fu+aL306SZ1YXPsuxtWNChVvOV9LS6a9V5HRf8Egv+TObP\\\/sOX380r50+gPd\\\/2xv+TUPjB\\\/2Kep\\\/+kz0Aflj+yd+zT8R\\\/id8EdI13w14d\\\/tHS5ri5RJ\\\/ttvFkrKysNryKeo9K\\\/W8g4ky\\\/L8vhhq7akr7K+78j8rz\\\/ACDH4\\\/MJ18PFOLt1tsvM9g\\\/4Yo+M3\\\/Qnj\\\/waWf8A8er6H\\\/XHKf5pf+As+e\\\/1Tzb\\\/AJ9r\\\/wACX+Zqj9kX4xjSRZf8K500sFKfa\\\/t1p5\\\/Jzkt5+CeMA44H41n\\\/AK25Tzc3tZ+nK7Gv+q+a8nL7Jf8AgS\\\/zMr\\\/hij4zf9CeP\\\/BpZ\\\/8Ax6tP9ccpf2pf+AszXCmbL\\\/l2v\\\/Al\\\/mb7\\\/srfGSS2EJ+GOi8BgJRd2obnJzxcY78ccDiuNcU5SpX9pL\\\/wFm74YzRq3sl\\\/4Ev8zCT9iv4zRuG\\\/4Q1XAOdr6nZ4Psf31dn+uOUdJS\\\/8BZz\\\/AOqea\\\/8APtf+BL\\\/M1bn9kr4y3FtcxD4d6bEZ2ZvMjvbNWjJbOU\\\/f\\\/LjGPpxWX+tmUrarP\\\/wF\\\/wCRq+F82ens196\\\/zMgfsUfGYH\\\/kTx\\\/4M7P\\\/AOPVq+Mcp\\\/ml\\\/wCAsz\\\/1UzX\\\/AJ9r\\\/wACX+Zbu\\\/2N\\\/jJd21vEPAVrbmEEGWHUrUPLkD7+ZyO3bHU+tZQ4uyqLbc5O\\\/wDdZUuFc1aSVNf+BL\\\/Mqf8ADFHxm\\\/6E4f8Ag0s\\\/\\\/j1a\\\/wCuOU\\\/zS\\\/8AAWR\\\/qnm3\\\/Ptf+BL\\\/ADLd\\\/wDscfGW+8sL4EtrRUGNtvqVoA3uczk1lT4uyqC1nJ\\\/9usufCuayd1TS\\\/wC3l\\\/mVP+GKPjN\\\/0J4\\\/8Gln\\\/wDHq1\\\/1xyn+aX\\\/gLI\\\/1Tzb\\\/AJ9r\\\/wACX+Z5d+09+y38Tfh98CfFPiDX\\\/Di6fpNktuZ7g39tJt3XESL8qSFjlmA4HevBz7ibLswy2phqMnzSta6fRp\\\/oe9kfDuYYLMKeIrRSjG99V1TR9kf8Egv+TObP\\\/sOX380r8hP1c93\\\/AGxv+TUPjB\\\/2Kep\\\/+kz0AeN\\\/8Eml3fsVeF\\\/+whqH\\\/pS9J66Ct1PsJ0wpIzS5V\\\/VgscrD47s57qKFINQ2yMV81rGVEQ+YkYDFgMbjIMewY9BmjlX9WF\\\/XU2bq+e3jDCN5suq7YwCRlgC3JHAzk+wNHKv6sO39alqzl+0iQ9lYqPfBIzx9KOVf1YLFjyxRyr+rBYPLFHKv6sFg8sUcq\\\/qwWDyxRyr+rBYPLFHKv6sFg8sUcq\\\/qwWDyxRyr+rBY+XP+Cna7f2Gfid\\\/1z0\\\/\\\/ANONtTtYepxn\\\/BIL\\\/kzmz\\\/7Dl9\\\/NKYHu\\\/wC2N\\\/yah8YP+xT1P\\\/0megDxz\\\/gkz\\\/yZT4X\\\/AOwhqH\\\/pS9AH2HIgkQqehoAjNrGTnYM5zmgQfZYz\\\/D+poGPSNY84HJ60APoAKACgAoAKACgAoA+W\\\/wDgp5\\\/yYz8Tv+uen\\\/8ApxtqAOK\\\/4JBf8mc2f\\\/Ycvv5pQB7v+2N\\\/yah8YP8AsU9T\\\/wDSZ6APHP8Agkz\\\/AMmU+F\\\/+whqH\\\/pS9AH11qVu91aSxJGJPMUoymZouDwcMoJB9xVJ2dzOabi0vzsZ8mgRahFH9utVMiIsYK3LsQFOQd2Ac+\\\/Wk99BxulZ\\\/5kt34etb65E80bO4Ocee4XoB06dv1Pqcosml0iCaaGVoR5kW0IwlYbcHIHH+SODkUAV5fDFhPPNK9ojPNnzN0jENn1B4\\\/wD1UAPn8O2dyYd9vxDH5SKs7hQmCMYHBGD\\\/AC9BQA6x0K3066M8EW2QoUyZWbIJyck8nnnn1PrQBoZf+6v\\\/AH1\\\/9agAy\\\/8AdX\\\/vr\\\/61ABl\\\/7q\\\/99f8A1qAFUtnkAD2Of6UAfLv\\\/AAU8\\\/wCTGfid\\\/wBc9P8A\\\/TjbUAcV\\\/wAEgv8Akzmz\\\/wCw5ffzSgD3f9sb\\\/k1D4wf9inqf\\\/pM9AHjn\\\/BJn\\\/kynwv8A9hDUP\\\/Sl6APsKVmVGKgM2OATjJobsByeo\\\/EzRNHuRbX161vcFJZBGbOViyxyGNiNoOcFSf8AdG7pzXPQrwxEeaHmvubT\\\/FCTT2DVPiboejanHp9zf5upGCBILSaYB\\\/m+UlFPJ2tgdTg+ldAzf1XU10aza6uXIgQjcYrd5CMkDO1cnGTyccDJOACapRcnaJEpKCuyfT7xL+zhuI5FljlQSI6ggMpGQcHnoRU7FJ3VyzQMKACgAoAKACgD5b\\\/4Kef8mM\\\/E7\\\/rnp\\\/8A6cbagDiv+CQX\\\/JnNn\\\/2HL7+aUAe8ftjf8mofGD\\\/sVNT\\\/APSZ6APG\\\/wDgkz\\\/yZT4X\\\/wCwhq
```

Comment:
Given the absolute ginormous size of this line, I think it also justifies having its own fixture file as mentioned before – ALTHOUGH I would try testing it with the former fixture used by `resetGlobals` to see if this one is needed at all – my guess is that it isn't. I don't think there is a substantial difference in the global here that is unique to the settings page.